### DISCUSSION TITLE
Failed to build milvus on mac M1

### DISCUSSION THREAD

**Discussion by psc0606** [milvus-io/milvus#25236] [Announcement]
Environment: master branch && macOS Ventura && Apple M1 Pro
After run ` ./scripts/install_deps.sh`, I started to build milvus by running `make -j 8`, but I got error in attached log. 
[build.log](https://github.com/milvus-io/milvus/files/11903604/build.log)

**Comment by xiaofan-luan** [comment]
could you go to /Users/psc/.conan/data/rocksdb/6.29.5/_/_/build/5c780fb2490783e88df7e41813bee3baccb35f6b
and run make see the error?

**Reply by psc0606** [reply]
I just got a different error, 
![image](https://github.com/milvus-io/milvus/assets/7888889/b49e1c92-aa67-42dd-be91-e3d39cee1a41)

**Reply by yhmo** [reply]
@psc0606  
Did you run the setenv.sh?
`source scripts/setenv.sh`

**Reply by psc0606** [reply]
Yeah,  same error

**Reply by yhmo** [reply]
It is a compile error for rocksdb, you can search the keywords on Google. I guess it is related to your local environment.
